content/pages/sphinx.html

```diff
@@ -0,0 +1,294 @@
+        <div class="meta">
+          Game Engine
+        </div>
+        <div class="description">
+          Godot Engine is a free and open source community-driven 2D and 3D game engine.
+        </div>
+      </div>
+    </a>
+</div>
+
+<div class="ui very padded basic vertical segment">
+  <div class="ui horizontal divider">Try it out</div>
+</div>
+
+<div class="ui horizontally container">
+  <div class="ui very padded centered grid">
+    <div class="middle aligned row">
+
+      <h2 class="ui huge header">
+        Everything you need, and nothing you don't.
+      </h2>
+
+      <div class="ten wide computer sixteen wide tablet sixteen wide mobile column">
+        <div class="ui large left aligned basic vertical segment">
+
+          {# Stylized code blocks #}
+          {% call technical.code_block(title="<code>.readthedocs.yaml</code>", icon="fa-file") %}
+          {% markdown %}
+
+          ```yaml
+          version: 2
+          build:
+            os: ubuntu-22.04
+            tools:
+              python: "3.10"
+              # You can also specify other tool versions:
+              # nodejs: "16"
+          
+          # Build documentation in the docs/ directory with Sphinx
+          sphinx:
+             configuration: docs/conf.py
+          
+          # Requirements required to build your docs
+          python:
+             install:
+             - requirements: docs/requirements.txt
+          ```
+
+          {% endmarkdown %}
+          {% endcall %}
+        </div>
+      </div>
+
+      <div class="six wide computer twelve wide tablet sixteen wide mobile column">
+        <div class="ui padded basic segment">
+          <div class="ui relaxed list">
+
+            {%- set header_1 = "Create an account" %}
+            {%- set icon_1 = "fa-square-1" -%}
+            <span class="ui small teal header item">
+              <i class="fad {{ icon_1 }} secondary big icon"></i>
+              <span class="content">
+                {{ header_1 }}
+                <span class="sub header">Sign up with GitHub or your email.</span>
+              </span>
+            </span>
+
+            {%- set header_2 = "Import your project" %}
+            {%- set icon_2 = "fa-square-2" -%}
+            <span class="ui small header item">
+              <i class="fad {{ icon_2 }} secondary big icon"></i>
+              <span class="content">
+                {{ header_2 }}
+                <span class="sub header">Select your existing git repositories with a 1-click interface.</span>
+              </span>
+            </span>
+
+            {%- set header_3 = "Add YAML config" %}
+            {%- set icon_3 = "fa-square-3" -%}
+            <span class="ui small header item">
+              <i class="fad {{ icon_3 }} secondary big icon"></i>
+              <span class="content">
+                {{ header_3 }}
+                <span class="sub header">Copy this one, it probably does what you want :)</span>
```

Comment:
Not 100% on emoji here, there might be more required to make that work. See my example page for an emoji example.

```suggestion
                <span class="sub header">Copy this example, it probably does what you want :smile:</span>
```